filebeat/_meta/common.reference.p2.yml

```diff
@@ -258,6 +258,31 @@ filebeat.inputs:
   # The number of seconds of inactivity before a remote connection is closed.
   #timeout: 300s
 
+#------------------------------ Syslog input --------------------------------
+# Experimental: Config options for the Syslog input
+#- type: syslog
+  #enabled: false
+
+  #protocol.tcp:
+    # The host and port to receive the new event
+    #host: "localhost:9000"
+
+    # Character used to split new message
+    #line_delimiter: "\n"
+
+    # Maximum size in bytes of the message received over TCP
+    #max_message_size: 20MiB
+
+    # The number of seconds of inactivity before a remote connection is closed.
+    #timeout: 300s
+
+  #protocol.udp:
+    # The host and port to receive the new event
+    #host: "localhost:9000"
+
+    # Maximum size of the message received over UDP
+    #max_message_size: 10240
```

Comment:
Should we bring message size for the 2 in line? Also the way it's configured, I like the humanize one.

Reply:
I will need to add support for humanize in udp, I will do it.

Reply:
@kvch @ruflin done in https://github.com/elastic/beats/pull/6886